### PR TITLE
This endpoint creates webhook for pipeline using GitHub

### PR DESCRIPTION
**Pull request by JuanitoFatas**
The response is `201 Created` (no content).

### CODE REVIEW

File: pages/apis/rest_api/pipelines.md.erb [hunk 1]
```diff
@@ -565,7 +565,7 @@ Success response: `204 No Content`
 
 ## Add a webhook
 
-Create webhook for an existing pipeline.
+Create SCM webhook for an existing pipeline using GitHub.
```
Comment (blaknite) [inline pages/apis/rest_api/pipelines.md.erb:568]:
```suggestion
Create an SCM webhook for an existing pipeline. Currently supports pipelines configured using our GitHub App.
```

**Review by blaknite — APPROVED** [review]

**Review by yob — COMMENTED** [review]
neat, the extra detail will be useful!

I reckon it's worth waiting for @plaindocs docs to take a look too, just in case there's additional feedback he has.

**Review by plaindocs — CHANGES_REQUESTED** [review]
I think that with this change we can push it out.  

One other possiblity would be to change the title to mention GitHub, as it is the only option.

**Review by plaindocs — APPROVED** [review]

File: pages/apis/rest_api/pipelines.md.erb [hunk 2]
```diff
@@ -565,7 +565,7 @@ Success response: `204 No Content`
 
 ## Add a webhook
 
-Create an SCM webhook for an existing pipeline. Currently supports pipelines configured using our GitHub App.
+Create an Githook webhook for an existing pipeline that is configured using our GitHub App. Pushes to the linked GitHub repository will trigger builds.
```
Comment (plaindocs) [inline pages/apis/rest_api/pipelines.md.erb:568]: Pretty sure I meant to write GitHub here. :man_facepalming:  

```suggestion
Create an GitHub webhook for an existing pipeline that is configured using our GitHub App. Pushes to the linked GitHub repository will trigger builds.
```

**Pull request merged**